server/schemas/schemaController.js

```diff
@@ -2,19 +2,8 @@ var bufferUtils = require("../utils/buffer");
 var Busboy = require("busboy");
 var helpers = require("../config/helpers");
 var s3Cache = require("../db/s3Cache");
-
 var ASCII_NEWLINE = 10;
-
-/**
- * Validate an entry conforms to a schema.
- * @param {string} schemaName the schema name
- * @param {Object} header the header row buffer of the entry
- * @param {Object} callback the callback that handles the validation response
- */
-var validateEntry = function(schemaName, header, callback) {
-  // TODO: verify entry conforms to schema (#62)
-  callback(null, true);
-};
+var schemaMatcher = require("./schemaMatcher");
```

Comment:
looks like this may be a display bug? Check 'view the whole file' and let me know if the require statement isn't at the top.